TheFourFifths/consus
# HotFix: Adding Student info to overdue items.

### Summary

Added a section for the student name and ID so that UI can know who has the overdue items checked out. This should've been put into the call originally, but I didn't think about it at the time.

### Checklist


### How to Review

1. Pull
2. Run Consus
3. Navigate to http://localhost/api/item/overdue
4. Notice that the overdue item now has a property "student" with the name and id of boaty.

### Links

- [TFF-151](https://msoese.atlassian.net/browse/TFF-151) <- The task I need this fix for.
